virttest/qemu_vm.py

```diff
@@ -898,11 +898,17 @@ def add_memorys(devices, params):
             """
             options = []
             mem_params = params.object_params("mem")
-            if mem_params.get("maxmem"):
+            mem_params.setdefault("mem", "512")
+            automem = mem_params.get("automem", "yes") == "yes"
```

Comment:
if source VM memory size not equal to destination VM  memory size, migration maybe failed too, so  that's why https://github.com/autotest/tp-qemu/pull/933 disable automem  for migration test.